api/models/flaggedItem/notifyUtils.js

```diff
@@ -23,11 +23,22 @@ export async function notifyModeratorsMember (flaggedItem) {
 }
 
 export function sendToCommunities (flaggedItem, communities) {
+  var sendToAdmins = () => Promise.resolve()
```

Comment:
If you're just going to call this within a `.then`, it doesn't need to return a promise because it'll already be wrapped in one. Given that, I would probably rewrite this as:
```js
const shouldSend = ...
// ...
.then(() => shouldSend && sendToAdmins())
```
rather than redefining a function. it seems more straightforward to me that way.